test/integration/node-specific/bson-options/useBigInt64.test.ts

```diff
@@ -0,0 +1,234 @@
+import { expect } from 'chai';
+
+import { BSON, Collection, Db, MongoAPIError, MongoClient } from '../../../mongodb';
+import { setupDatabase } from '../../shared.js';
+
+describe('useBigInt64 option', function () {
+  let configuration;
+  beforeEach(function () {
+    configuration = this.configuration;
+    return setupDatabase(this.configuration);
+  });
+
+  describe('when not provided to client', async function () {
+    let client: MongoClient;
+
+    beforeEach(async function () {
+      client = configuration.newClient(configuration.writeConcernMax());
+    });
+
+    afterEach(async function () {
+      await client.close();
+    });
+
```

Comment:
We could consider moving the `client` / `db` / `collection` variables up to the top-level describe, and then having a single `afterEach` cleanup hook.  There's a lot of duplication in cleanup